.gitignore

```diff
@@ -7,6 +7,7 @@ credentials.properties
 # Gradle
 .gradle
 build
+rskj-core/src/main/resources/build-info.properties
```

Comment:
This should not be generated in this location anymore. Could you please check in your local environment if just removing it keeps everything working?

Reply:
It does. Removed.